GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -29,6 +29,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+// Parameters for the auth and token exchange endpoints.
+static NSString *const kAudienceParameter = @"audience";
```

Comment:
These should be declared with `extern` in the header file of the constants class that you make. Then, you will provide the definition in the `.m`.

https://google.github.io/styleguide/objcguide.html#constants

Reply:
Followed suggestion and made new constants class.